internal/dwarf/frame/table.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/parca-dev/parca-agent/internal/dwarf/util"
 )
 
+// TODO(javierhonduco): Add source
+const RBPRegister = 6
```

Comment:
Better to use https://github.com/go-delve/delve/tree/master/pkg/dwarf/regnum

Reply:
Thanks so much! Did not look in the right places